clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/jobhandling/JobHandlerInvokingSpringBeans.java

```diff
@@ -116,7 +117,7 @@ private List<Object> createParameters(final JobClient jobClient, final Activated
     return parameterResolvers.stream().map(resolver -> resolver.resolve(jobClient, job)).toList();
   }
 
-  private FinalCommandStep createCompleteCommand(
+  private FinalCommandStep<CompleteJobResponse> createCompleteCommand(
```

Comment:
The missing generic type declaration here was the cause for the compiler to not raise the conflict already when the CommandWrapper for the Complete Command is created [here](https://github.com/camunda/camunda/blob/main/clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/jobhandling/JobHandlerInvokingSpringBeans.java#L92)